src/EditorFeatures/Core/Implementation/SmartIndent/SmartIndent.cs

```diff
@@ -10,6 +10,10 @@
 using Microsoft.VisualStudio.Text.Editor;
 using Roslyn.Utilities;
 
+using NewIndentationService = Microsoft.CodeAnalysis.Indentation.IIndentationService;
+using OldIndentationService = Microsoft.CodeAnalysis.Editor.IIndentationService;
```

Comment:
why are we keeping this old indentation service stuff? what is this synchronousIndentationservice anyway?